pkg/arm/arm.go

```diff
@@ -77,10 +83,133 @@ func (g *simpleGenerator) Generate(ctx context.Context, cs *api.OpenShiftManaged
 		return nil, err
 	}
 
-	var azuretemplate map[string]interface{}
-	err = json.Unmarshal(azuredeploy, &azuretemplate)
-	if err != nil {
+	var original, copied map[string]interface{}
+	if err := json.Unmarshal(azuredeploy, &original); err != nil {
+		return nil, err
+	}
+	if err := json.Unmarshal(azuredeploy, &copied); err != nil {
+		return nil, err
+	}
+	if err := hashScaleSets(original, copied); err != nil {
 		return nil, err
 	}
-	return azuretemplate, nil
+	return original, nil
+}
+
+func hashScaleSets(original, copied map[string]interface{}) error {
+	for key, value := range copied {
+		if key != "resources" {
+			continue
+		}
+
+		for _, r := range value.([]interface{}) {
+			resource, ok := r.(map[string]interface{})
+			if !ok {
+				continue
+			}
+
+			if !isScaleSet(resource) {
+				continue
+			}
+
+			// cleanup capacity
+			deleteField(resource, "sku", "capacity")
```

Comment:
Just for my info, why we cleaning this one?

Reply:
So that scaling up/down will not cause unnecessary VM rotations.